webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    filename: 'awesome.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        use: [
+          'style-loader',
+          'css-loader',
+          'sass-loader',
+        ],
+      },
+    ],
+  },
+  devServer: {
+    contentBase: path.join(__dirname, 'public'),
+    port: 9000,
+  },
+};
```

Comment:
- [ ] Kindly make sure the webpack is configured perfectly. As when I try to run your server this is the error I am getting.
![Screenshot from 2022-07-28 00-09-14](https://user-images.githubusercontent.com/83042742/181381184-24f9c341-a76c-4835-9d6b-b208d0aaed40.png)
**But to solve the issue you can go to this **[article](https://stackoverflow.com/questions/67926476/webpack-dev-server-config-contentbase-not-working-in-latest-version)** and you will find an explanation about fixing your issue.**